pip/rapids-dask-dependency/pyproject.toml

```
@@ -15,6 +15,7 @@ dependencies = [
     "dask @ git+https://github.com/dask/dask.git@main",
     "distributed @ git+https://github.com/dask/distributed.git@main",
 ]
+license = { text = "Apache 2.0" }
```

This would make it [compatible with SPDX]( https://spdx.org/licenses/Apache-2.0.html )

```suggestion
license = { text = "Apache-2.0" }
```

---

Our license checker requires this to be `Apache 2.0` without the hyphen, so I've reverted this change in 6d271fe